README.md

```diff
@@ -35,10 +34,17 @@ go get -u github.com/haya14busa/gopkgs/cmd/gopkgs
 ```
 $ gopkgs -h
 Usage of gopkgs:
-  -fullpath
-        output absolute file path to package directory. ("/usr/lib/go/src/net/http")
-  -short
-        output vendorless import path ("net/http", "foo/bar/vendor/a/b")
+  -f string
+    	output format of the package (default "{{.ImportPathShort}}")
+
+
+Use -f to custom the output using template syntax. The struct being passed to template is:
+    type Pkg struct {
+        Dir             string // absolute file path to Pkg directory ("/usr/lib/go/src/net/http")
+        Name            string // package name ("http", "a")
```

Comment:
Shouldn't this be "b" to be consistent with the comments for ImportPath and ImportPathShort?

Or, I prefer completely remove "a" example. It doesn't help us understand what "Name" means.

Reply:
Agree